mypyc/test-data/run-classes.test

```diff
@@ -1636,3 +1636,56 @@ from native import A, B, C
 a = A()
 b = B()
 c = C()
+
+[case testAttrOverrides]
+from typing import Any
+
+class Proxy:
+    def __setattr__(self, name:str, value: Any) -> None:
+        print("__setattr__")
+
+    def __getattr__(self, name:str) -> Any:
+        print("__getattr__")
+
+    def __setitem__(self, key: Any, value: Any) -> None:
+        print("__setitem__")
```

Comment:
Use `int` for key and value arguments to test argument unboxing.

In addition to the `None` return type, test the `Any` return type.